modules/images-s2i-java-pulling-images.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/using-images-source-to-image.adoc
+
+[id="images-s2i-java-pulling-images_{context}"]
+= Pulling images for Java
+
+The Red Hat Enterprise Linux (RHEL) 8 image is available through the Red Hat Registry.
```

Comment:
OCP 4.6 is now fully based on RHEL 8. We should encourage customers to use RHEL 8 images to the furthest extent possible.

Note that I think this image is also available as an ImageStream installed by the Samples Operator.